bitwuzla/default.nix

```diff
@@ -0,0 +1,79 @@
+# This is adapted from https://github.com/NixOS/nixpkgs/pull/246779, which has
+# not yet been merged upstream into nixpkgs.
+
+{ pkgs
+, stdenv ? pkgs.stdenv
+, fetchFromGitHub ? pkgs.fetchFromGitHub
+, lib ? pkgs.lib
+, python3 ? pkgs.python3
+, meson ? pkgs.meson
+, ninja ? pkgs.ninja
+, git ? pkgs.git
+, gmp ? pkgs.gmp
+, gtest ? pkgs.gtest
+, cadical ? pkgs.cadical
+, symfpu ? pkgs.symfpu
+, kissat ? pkgs.kissat
+, cryptominisat ? pkgs.cryptominisat
+, pkg-config ? pkgs.pkg-config
+, writeTextDir ? pkgs.writeTextDir
```

Comment:
This part was cargo-culted from https://github.com/NixOS/nixpkgs/pull/246779. There's a lot of duplication here, so perhaps I ought to simply remove these as arguments to the derivation?